<launch>
  <!-- interface -->
  <arg name="input/from_can" default="~/input/from_can" />
  <arg name="output/to_can" default="~/output/to_can" />
  <!--<arg name="input/from_can" default="/from_can_bus1" />
  <arg name="output/to_can" default="/to_can_bus1" />-->
  <arg name="output/objects" default="~/output/objects" />
  <arg name="output/scan" default="~/output/scan" />
  <!-- parameter -->
  <arg name="output_frame" default="front_radar" />
  <arg name="publish_radar_track" default="true" />
  <arg name="publish_radar_scan" default="true" />
  <arg name="sequential_publish" default="false" />
  <arg name="size_x" default="1.8" />
  <arg name="size_y" default="1.8" />

  <node pkg="tf2_ros" exec="static_transform_publisher" name="map_to_front_radar_link_broadcaster" args="0 0 0 0 0 0 map front_radar" output="screen"/>

  <node pkg="ars408_ros" exec="ars408_node" name="ars408_node" output="screen">
    <remap from="~/input/from_can" to="$(var input/from_can)" />
    <remap from="~/output/to_can" to="$(var output/to_can)" />
    <remap from="~/output/objects" to="$(var output/objects)" />
    <remap from="~/output/scan" to="$(var output/scan)" />
    <param name="publish_radar_track" value="$(var publish_radar_track)" />
    <param name="publish_radar_scan" value="$(var publish_radar_scan)" />
    <param name="output_frame" value="$(var output_frame)" />
    <param name="sequential_publish" value="$(var sequential_publish)" />
    <param name="size_x" value="$(var size_x)" />
    <param name="size_y" value="$(var size_y)" />
  </node>
</launch>
